scripts/run-e2e-tests.sh

```diff
@@ -87,7 +87,8 @@ function install_cluster() {
   if [ "${GIT_CLONE}" = "true" ]; then
     check_cmd "git"
     rm -rf ./build/yunikorn-release
-    git clone --depth 1 https://github.com/apache/yunikorn-release.git ./build/yunikorn-release
+    # TODO: update following branch if PR in yunikorn-release is merged
+    git clone --depth 1 https://github.com/FrankYang0529/yunikorn-release.git -b YUNIKORN-2135 ./build/yunikorn-release
```

Comment:
As indicated in the yunikorn-release repo PR, I think we should find a way to deploy this without polluting the upstream helm charts.

Reply:
Here's the approach we should use: https://helm.sh/docs/topics/advanced/#post-rendering

Helm supports a post-renderer that can be used to update the generated output. We can use this in the e2e test script during install to "patch" the installed charts with what is required here. An ideal post-renderer would be kustomize (see https://kustomize.io/). An example using the two together can be found here: https://github.com/thomastaylor312/advanced-helm-demos/tree/master/post-render

We can setup a kustomize script to patch only the objects we need so that the original helm charts can stay pristine. All that should be required is an ENV var and build-mount for /tmp/coverage (or something like it).